### PR TITLE
Features/101

### PR DESCRIPTION
**Pull request by sorobon**
### Description of the Change

Debian 11 support added
Debian 8 support removed
Molecule/ansible updated

### Applicable Issues

[#101 ]

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,26 @@
 ---
 dist: bionic
```
Comment (vicsufer) [inline .travis.yml]: Should we use focal instead (ubuntu 20)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/nginx_role/tree/develop)
+### Added
+- *[#101](https://github.com/idealista/nginx_role/issues/101) Debian 11 support* @sorobon
+### Removed
+- Debian 8 (jessie) support
```
Comment (vicsufer) [inline CHANGELOG.md]: We should also link "[#101]" issue I guess, or at least the PR number (102)

**Review by vicsufer — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (sorobon) [reply .travis.yml]: done

File: CHANGELOG.md (hunk 2 above)
Comment (sorobon) [reply CHANGELOG.md]: done

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ These instructions will get you a copy of the role for your ansible playbook. On
 
 ### Prerequisities
 
-Ansible 2.9.x.x version installed.
+Minimun ansible 2.9.x.x version installed.
```
Comment (xtianae7) [inline README.md]: typo :)

**Review by xtianae7 — APPROVED** [review]

File: .travis.yml [hunk 4]
```diff
@@ -1,20 +1,26 @@
 ---
-dist: bionic
+dist: focal
 language: python
-python: "3.7"
+python: "3.9"
```
Comment (ablancolopez) [inline .travis.yml:4]: should we start testing on Python 3.10?

**Review by ablancolopez — APPROVED** [review]

**Review by jperera — APPROVED** [review]

**Review by emepege — APPROVED** [review]

File: .travis.yml (hunk 4 above)
Comment (sorobon) [reply .travis.yml:4]: yeah :)

File: README.md (hunk 3 above)
Comment (sorobon) [reply README.md]: done

**Review by frantsao — APPROVED** [review]

**Pull request merged**